examples/step-87/doc/results.dox

```diff
@@ -0,0 +1,269 @@
+  - write mesh (immersed mesh)
+
+time: 0.01
+  - move support points (immersed mesh)
+  - compute to be tested values (immersed mesh)
+  - test values (background mesh)
+
+time: 0.02
+  - move support points (immersed mesh)
+  - compute to be tested values (immersed mesh)
+  - test values (background mesh)
+
+...
+
+time: 2
+  - move support points (immersed mesh)
+  - compute to be tested values (immersed mesh)
+  - test values (background mesh)
+  - write data (background mesh)
+  - write mesh (immersed mesh)
+@endverbatim
+
+The output visualized in Paraview looks like the following: The deformation of the
+immersed mesh by the reversible vortex flow can be seen. Due to discretization
+errors, the circular shape is not exactly reobtained at the end, illustrated in
+the right figure. The sharp nature of the surface-tension force vector, shown as
+vector plots, can be seen by its restriction to cells that are intersected by the
+immersed mesh.
+
+<table align="center" class="doxtable">
+  <tr>
+    <td>
+        @image html step_87_ex_3_force.0000.png
+    </td>
+    <td>
+        @image html step_87_ex_3_force.0010.png
+    </td>
+    <td>
+        @image html step_87_ex_3_force.0020.png
+    </td>
+  </tr>
+</table>
+
+<h3>Possibilities for extension</h3>
+
+This program highlights some of the main capabilities
+of the distributed evaluation routines in deal.II. However, there are many related
+topics worth mentioning:
+- Performing a distributed search is an expensive step. That is why we suggest
+to provide hints to Utilities::MPI::RemotePointEvaluation and to reuse Utilities::MPI::RemotePointEvaluation
+instances in the case that the communication pattern has not changed. Furthermore,
+there  are instances where no search is needed and the points are already sorted
+into the right cells. This is the case if the points are generated on the cell level
+(see step-85; CutFEM) or the points are automatically sorted into the correct (neighboring) cell
+(see step-68; PIC with Particles::ParticleHandler). Having said that, the Particles::ParticleHandler might use
+the described infrastructure to perform the initial sorting of particles into
+cells.
+- We concentrated on parallelization aspects in this tutorial. However, we would
+like to point out the need for fast evaluation on cell level.
+The task for this in deal.II is FEPointEvaluation. It exploits the structure of
+@f[
+\hat{u}(\hat{\boldsymbol{x}}) = \sum_i \hat{N}_i(\hat{\boldsymbol{x}}) \hat{u}_i
+@f]
+to derive fast implementations, e.g., for tensor-product elements
+@f[
+\hat{u}(\hat{x}_0, \hat{x}_1, \hat{x}_2) =
+\sum_k \hat{N}^{\text{1D}}_k(\hat{x}_2)
+\sum_j \hat{N}^{\text{1D}}_j(\hat{x}_1)
+\sum_i \hat{N}^{\text{1D}}_i(\hat{x}_0)
+\hat{u}_{ijk}.
+@f]
+Since only 1D shape functions need to be tabulated, this formulation is faster than
+the original one but still slower than as if the tensor-product structure of
+evaluation points would be exploited as in the case of FEEvaluation.
```

Comment:
```suggestion
evaluation points would be exploited as in the case of FEEvaluation.
```
would be or was exploited?

Reply:
This should be addressed by reformulation of the entire sentence.